### ISSUE TITLE
mswin build was failed with the current master branch

### ISSUE THREAD

**Issue by hsbt** [rubygems/rubygems#3121]
The current master branch (https://github.com/rubygems/rubygems/commit/218ca18c3b3ae84fd1eae3b8d5270fb0b537a0c3) was failed with mswin CI environment of ruby core team. 

https://rubyci.org/logs/mswinci.japaneast.cloudapp.azure.com/vc12-x64/ruby-master/log/20200202T213541Z.fail.html.gz

I'm not sure why it was broken yet. 

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

**Comment by MSP-Greg** [comment]
Interesting.

Working on some Actions stuff, the following builds passed:
```
ruby 2.8.0dev (2020-02-02T14:09:25Z master fae537259d) [x64-mswin64_140]
ruby 2.8.0dev (2020-02-03T01:27:09Z master d9552cbd5a) [x64-mswin64_140]
```

Also, I just rebased https://github.com/ruby/ruby/pull/2838 with a new Actions mswin workflow, and the rebase included fae5372, see:
https://github.com/ruby/ruby/runs/422207059

Seems like one of those "can't repo" runs?

**Comment by hsbt** [comment]
This issue was resolved now.
